structures_and_records.md

```diff
@@ -49,7 +49,8 @@ structure Point (α : Type u) where
 Values of type ``Point`` are created using ``Point.mk a b``, and the
 fields of a point ``p`` are accessed using ``Point.x p`` and
 ``Point.y p``. The structure command also generates useful recursors and
-theorems. Here are some of the constructions generated for the
+theorems. Dot notation is also supported so you can write `p.x`
```

Comment:
Duplicate of line 95ff

Reply:
True, and good point, but I felt uneasy with ``Point.x p`` and wanted to be rescued by the dot notation sooner :-)